examples/hooks/group_hooks.py

```diff
@@ -19,19 +19,25 @@ async def third_hook(ctx: crescent.Context):
     print("Here third.")
 
 
-group = crescent.Group("my_group", hooks=[first_hook])
-sub_group = group.sub_group("my_sub_group", hooks=[second_hook])
+group = crescent.Group("my_group", hooks=[first_hook], hook_after=[first_hook])
+sub_group = group.sub_group("my_sub_group", hooks=[second_hook], hooks=[second_hook])
```

Comment:
Defined `hooks` twice? Also why didn't this test fail, you can't define kwargs twice...

Reply:
Oh nvm it's an example lol

Still should fix